<launch>
    <node pkg="hanoying_back" type="sim_to_raw.py" name="sim_to_raw"/>

    <node pkg="hanoying_back" type="game_state.py" name="game_state" output="screen"/>
    <node pkg="hanoying_back" type="game_solve.py" name="game_solve" output="screen"/>
    <node pkg="hanoying_back" type="game_move.py" name="game_move" output="screen"/>
    <node pkg="hanoying_back" type="decision_system.py" name="decision_system" output="screen"/>
</launch>
